internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -264,6 +266,8 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
           .put(SERVER_INGESTION_HEARTBEAT_INTERVAL_MS, 5000)
           .put(SERVER_LEADER_COMPLETE_STATE_CHECK_IN_FOLLOWER_VALID_INTERVAL_MS, 5000)
           .put(SERVER_RESUBSCRIPTION_TRIGGERED_BY_VERSION_INGESTION_CONTEXT_CHANGE_ENABLED, true)
+          .put(ROCKSDB_BLOCK_CACHE_SIZE_IN_BYTES, 8 * 1024 * 1024 * 1024L)
```

Comment:
Why do we need so big cache for integration tests and why we override this config in other integration tests?
Shall we just change it to be something like 256MB here and it will be good by default?

Reply:
Sure, but venice code has been using this much cache in tests currently.